### PR TITLE
D8 un 360 b

### PR DESCRIPTION
**Pull request by DuttonMa**
- Bug fix to Drupal console 
- Pull latest version of date_facet_drilldown module as it correctly handles active year facets when duplicated by the facets_summary module

### CODE REVIEW

File: .lando.base.yml [hunk 1]
```diff
@@ -35,6 +35,8 @@ services:
     hogfrom:
       - appserver
 tooling:
+  drupal:
+    cmd: "/app/vendor/bin/drupal --root=/app/web"
```
Comment (DuttonMa) [inline .lando.base.yml:39]: I know that the Drupal console is being phased out - but we still have some Drupal console commands in Unity so don't want it to go just yet !

**Review by soda736 — APPROVED** [review]
LG

**Review by omahm — APPROVED** [review]
LG, I thought lando had Drupal console still as part of the recipe

**Comment by DuttonMa** [comment]
> LG, I thought lando had Drupal console still as part of the recipe

Yes it does, but it wasn't working - this is a bug fix that I put in so that 'lando drupal' works correctly

**Pull request merged**